pipelines/ko-build-oci-ta/README.md

```diff
@@ -99,6 +99,7 @@ This pipeline is pushed as a Tekton bundle to [quay.io](https://quay.io/reposito
 |shortCommitLength| Length of short commit SHA| 7| |
 |sparseCheckoutDirectories| Define the directory patterns to match or exclude when performing a sparse checkout.| | |
 |sslVerify| Set the `http.sslVerify` global git config. Setting this to `false` is not advised unless you are sure that you trust your git remote.| true| |
+|submodulePaths| Comma-separated list of specific submodule paths to initialize and fetch. Only submodules in the specified directories and their subdirectories will be fetched. Empty string fetches all submodules. Parameter "submodules" must be set to "true" to make this parameter applicable. | | |
```

Comment:
@avi-biton any clues how we get this as part of our update? Do we still need this update or can the build-definitions rpms task can be deprecated in favor of the one released directly from upstream?